src/test/java/test/parameters/ParameterInjectAndOptionTest.java

```diff
@@ -0,0 +1,15 @@
+package test.parameters;
+
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+
+public class ParameterInjectAndOptionTest extends SimpleBaseTest {
+
+    @Test
+    public void test() {
```

Comment:
We should be adding an assertion here. Maybe add a listener and check to ensure that there are no failed methods perhaps ?

Reply:
You are right. Fixed it.